src/test/resources/application.conf

```diff
@@ -0,0 +1 @@
+identity.api.host="https://idapi.code.dev-theguardian.com"
```

Comment:
We should also be testing against idapi.code.dev-guardianapis.com and idapi.guardianapis.com.